backend/flask/app.py

```diff
@@ -318,6 +325,27 @@ def transaction_to_json(transaction):
         data['total_percent'] = category_sum
     return result
 
+
+"""
+helper function to query database and find rows matching the city, state, and salary fields of a user but not id, this 
+keeps the user from being compared to themselves
+"""
+def find_similar_users(profileData):
```

Comment:
excited to see this start to be implemented. This is definitely an area for potential iteration to make the likeness algorithm more complex.